### PR TITLE
Test cases for agencyComponentField.js

### PR DESCRIPTION
**Pull request by sunilb0575**
#### Which User story does this relate to?
-

#### Brief feature description
- Improve test coverage for agencyComponentField.js

#### Does this PR block/effect other stories?
-No

#### Do any commands needs to be run in order to run the project with this PR?
-No

#### Code coverage percent diff as a result of this PR?
-

### CODE REVIEW

**Comment by ratneshraval** [comment]
Can one of the admins verify this patch?

**Comment by ratneshraval** [comment]
@sunilb0575 what is the java update in gemfile.lock?

File: config/karma.conf.js [hunk 1]
```diff
@@ -95,22 +95,22 @@ module.exports = function (config) {
 
     // start these browsers
     // available browser launchers: https://npmjs.org/browse/keyword/karma-launcher
-    browsers: ['ChromeHeadless'],
+    browsers: ['Chrome'],
```
Comment (ratneshraval) [inline config/karma.conf.js]: please revert other changes except `functions: 65`

File: test/javascript/components/rfa_forms/agencyComponentFields_tests.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  beforeEach(() => {
+    q1History = {
+      'agencies': ['new']
+    }
+
+    removeAgencyCardSpy = jasmine.createSpy('removeAgencyCard')
+    onAgencyChangeSpy = jasmine.createSpy('onAgencyChange')
+
+    agencyComponent = shallow(<AgencyComponent
+      index={0}
```
Comment (ratneshraval) [inline test/javascript/components/rfa_forms/agencyComponentFields_tests.js]: if you are using value 0 everywhere may be declare it as a `let/const` and reuse it.

File: test/javascript/components/rfa_forms/agencyComponentFields_tests.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+import {AgencyComponent} from 'components/rfa_forms/agencyComponentFields.js'
+import {shallow, mount} from 'enzyme'
+
+describe('verify agency component fields ', () => {
+  let agencyComponent, onAgencyChangeSpy,
+    removeAgencyCardSpy, q1History, optionList
+
+  optionList = ['a', 'b', 'c']
```
Comment (ratneshraval) [inline test/javascript/components/rfa_forms/agencyComponentFields_tests.js]: is this for some dropdown? If so, we have a constant file for all test related dropdowns. You can use that file by adding/reusing from that and get better values than a,b,c.

**Comment by sunilb0575** [comment]
@ratneshraval addressed all your comments.

File: test/javascript/components/rfa_forms/agencyComponentFields_tests.js [hunk 4]
```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+import {AgencyComponent} from 'components/rfa_forms/agencyComponentFields.js'
+import {shallow, mount} from 'enzyme'
+import {stateTypes} from '../../helpers/constants'
+
+describe('verify agency component fields ', () => {
+  let agencyComponent, onAgencyChangeSpy,
+    removeAgencyCardSpy, q1History, optionList
+  let value = 0
```
Comment (ratneshraval) [inline test/javascript/components/rfa_forms/agencyComponentFields_tests.js]: Probably a good idea to use `indexValue`

**Pull request merged**